src/pheval/utils/phenopacket_utils.py

```diff
@@ -477,6 +478,59 @@ def diagnosed_variants(self) -> List[GenomicVariant]:
                 variants.append(variant)
         return variants
 
+    def check_incomplete_variant_record(self) -> bool:
+        """
+        Check if any variant record in the phenopacket has incomplete information.
+
+        This method iterates through the diagnosed variant records and checks if any of them
+        have missing or incomplete information such as empty chromosome, position, reference,
+        or alternate allele.
+
+        Returns:
+            bool: True if any variant record is incomplete, False otherwise.
+        """
+        variants = self.diagnosed_variants()
+        for variant in variants:
```

Comment:
Is this the case in the extended LIRICAL dataset? Isn't it just the case that the diagnosed variants should be empty?

Reply:
No, as it is still able to access the objects it returns these values when I access them in with Python, so still creates a list, just with empty string values